packages/create-block/lib/init-wp-scripts.js

```diff
@@ -21,22 +22,28 @@ module.exports = async function( {
 
 	info( '' );
 	info( 'Creating a "package.json" file.' );
-	await writePkg( cwd, {
-		name: slug,
-		version,
-		description,
-		author,
-		license,
-		main: 'build/index.js',
-		scripts: {
-			build: 'wp-scripts build',
-			'format:js': 'wp-scripts format-js',
-			'lint:css': 'wp-scripts lint-style',
-			'lint:js': 'wp-scripts lint-js',
-			start: 'wp-scripts start',
-			'packages-update': 'wp-scripts packages-update',
-		},
-	} );
+	await writePkg(
+		cwd,
+		omitBy(
+			{
+				name: slug,
+				version,
```

Comment:
Should we be wary of potentially omitting required fields? Is it possible?

https://docs.npmjs.com/creating-a-package-json-file#required-name-and-version-fields

Reply:
I agree with you @aduth, and I was about to flag it. However, given that we set some [defaults](https://github.com/WordPress/gutenberg/blob/master/packages/create-block/lib/templates.js#L13), I wasn't able to create empty values and for that reason, "break" the `readme.txt` file nor the `package.json`. Because of that, I ended up not commenting.

Reply:
Yes, both `name` and `version` are mandatory and they have corresponding validators in place. In fact, I added a validator for the `version` in this PR and it uses the regular expression recommended on the semver.org website. I'll include a note in CHANGELOG.